A/Argp/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Argp"
+version = v"1.4.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/ericonr/argp-standalone.git", "743004c68e7358fb9cd4737450f2d9a34076aadf"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/argp-standalone
+
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
```

Comment:
Are these patches upstreamed or going to be upstreamed? It would be great if they can be incorporated into upstream so we don't have to continually carry them and update them on new versions.

Reply:
I think this library has not undergoing any development for the last 10 years; one of the patches is for compilation on mingw. I don't plan to upstream them, if that's not too much of a disappointment to you :)